src/lib/integrations/daoStack/arc/schemes.ts

```diff
@@ -45,6 +45,21 @@ export const schemes: Scheme[] = [
       return [votingMachineParametersKey, votingMachineAddress]
     },
   },
+  {
+    typeName: "GenericScheme",
+    displayName: "DAO can call external contracts",
+    description:
+      "Proposals that, when passed, can call external functions in a generic manner.",
+    toggleDefault: true,
+    permissions: "0x00000000" /* no permissions */,
+    params: [],
+    getCallableParamsArray: function(
+      votingMachineParametersKey,
+      votingMachineAddress
+    ) {
+      return [votingMachineParametersKey, votingMachineAddress]
```

Comment:
It seems like this is also expecting the contract to call. See; https://github.com/daostack/arc/blob/eb0ca46d82828762650ca6875ffe6809001a4b94/contracts/universalSchemes/GenericScheme.sol#L117.

So I think it might need to be added one GenericScheme per contract that the DAO should be able to call(?).

Reply:
Yeah that was the trouble, my idea was that it would defer to the correct contract call when someone creates a proposal, rather than upon registration in this view which only seems to configure the voting machine. I'll comment this out for now.